### PR TITLE
removed spiffxp and added xmcqueen per nomination sig-testing chair

### PR DESCRIPTION
**Pull request by xmcqueen**
getting the sig lists up-to-date with chair rotation per kubernetes/community#7239

### CODE REVIEW

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

/hold
please cancel the hold at your discretion

File: config/kubernetes-sigs/sig-testing/teams.yaml [hunk 1]
```diff
@@ -31,16 +31,16 @@ teams:
     - alejandrox1
     - andrewsykim
     - BenTheElder
-    - spiffxp
     - vladimirvivien
+    - xmcqueen
```
Comment (BenTheElder) [inline config/kubernetes-sigs/sig-testing/teams.yaml:35]: This one and the others below in this file are actually subproject owners unrelated to the chair role.
We probably need to review these but I think that's a different PR.

**Comment by BenTheElder** [comment]
one nit 😅

File: config/kubernetes-sigs/sig-testing/teams.yaml (hunk 1 above)
Comment (mrbobbytables) [reply config/kubernetes-sigs/sig-testing/teams.yaml:35]: @BenTheElder to unblock should we move forward with merging this with a follow up to clean up? or @xmcqueen do you have time to update the PR?

Comment (BenTheElder) [reply config/kubernetes-sigs/sig-testing/teams.yaml:35]: The file above has the required change and all of the changes in this file should really be a separate cleanup, this PR alters some of the sig subprojects but not all and I think that was just a find + replace.

I don't think that was intended and I don't think we should merge that way. If Aaron wants to drop from these groups that's a different PR

**Comment by xmcqueen** [comment]
OK I finally noticed this PR is waiting for me to act.  I'll get it going.  Thanks

**Comment by xmcqueen** [comment]
I think I have restored spiffxp to all the proper places.  The PR is smaller now.

**Pull request merged**